conda-store-server/conda_store_server/_internal/worker/app.py

```diff
@@ -72,6 +72,7 @@ def start(self):
         argv = [
             "worker",
             "--loglevel=INFO",
+            "--max-tasks-per-child=10",  # mitigate memory leaks
```

Comment:
Looks good to me. I am assuming there is no limit by default.

Reply:
Yes, that's right.  By default, the workers will never restart due to finishing a certain number of tasks.